PeanutPlug.groovy

```diff
@@ -26,115 +26,46 @@
  *  2019-06-14 - V02.50 Added user variables Power Change Report Value, Power Reporting Interval,
  *                      Current Change Report Value, Current Reporting Interval, Voltage Reporting Interval, and
  *                      Debug Logging?
+ *  2019-06-25 - V02.60 Removed deprecated ST functions and tiles.  Enabled Energy reporting (in W*h). *

```

Comment:
I now realize that I didn't identify which changes are mine.  I will correct this.
Please indicate that this change is yours.

Reply:
I didn't see how you wanted to display the contributor of the change.  Will you just insert mine, following your lead?

Reply:
Hi Tom,
Actually, I was hoping you would know what the standard for that was.  I guess something like

  *  2019-06-14 - V02.50 JIMcGhee - Added user variables Power Change Report Value, Power Reporting Interval,
  *                      Current Change Report Value, Current Reporting Interval, Voltage Reporting Interval, and
  *                      Debug Logging?

What do you think?
Jim

Reply:
That looks good to me.  The only concern I would see if that all of the changes that pre-date you and me won't have author info.  I guess it is safe to assume they all came from pakmanwg, but at least one mentions a different person's name.  Since you originated this port, I think it is your call on how to format the attribution. :)  You have already done a more than sufficient job, in my opinion, by keeping the namespace and other original author information intact.  Perhaps you could email him/her to see whether any other notes are necessary and whether they have the author history on the other changes (?)